plugins/UsersManager/templates/index.twig

```diff
@@ -178,6 +178,7 @@
                 {% if showLastSeen is defined and showLastSeen %}
                 <th>{{ 'UsersManager_LastSeen'|translate }}</th>
                 {% endif %}
+                <th>token_auth</th>
```

Comment:
Yeah, that part is a rather ugly one. Personally I would like to completely separate the token management from the user management, like just having a separate `piwik-content-block` only listing the tokens and the regenerate buttons next to them.

The warning message for passwords should not feel to confusing having separate blocks. From a responsive layout perspective it might also be easier on the eyes to have those long strings separated if there are any plans to go like below 540px for everything (currently the users block needs like 9xx pixels and displaying the data veritcally (each cell as a row) might be nicer with less data per user).